frontend/javascripts/oxalis/view/action-bar/starting_job_modals.tsx

```diff
@@ -857,8 +860,15 @@ function StartJobForm(props: StartJobFormProps) {
       }
       if (jobCreditCostPerGVx != null && activeUser?.organization) {
         // As the job did cost credits, refetch the organization to have a correct credit balance.
-        const updatedOrganization = await getOrganization(activeUser?.organization);
-        dispatch(setActiveOrganizationAction(updatedOrganization));
+        try {
+          const updatedOrganization = await getOrganization(activeUser?.organization);
+          dispatch(setActiveOrganizationAction(updatedOrganization));
+        } catch (error) {
+          Toast.error(
+            "Failed to fetch updated amount of credits of your organization. See the console for more details. This usually shouldn't impact you.",
```

Comment:
this is too technical in my opinion. how about "There was an error while reloading the available credits. Consider reloading the page."

Reply:
Sure, sounds good. Thanks for the suggestion :pray: